picard/ui/options/cover.py

```diff
@@ -58,9 +58,9 @@ class CoverOptionsPage(OptionsPage):
         TextOption("setting", "cover_image_filename", "cover"),
         BoolOption("setting", "save_images_overwrite", False),
         ListOption("setting", "ca_providers", [
+            ('CaaReleaseGroup', True),
```

Comment:
Instead I would just set it to `True` but keep its position. This way cover art specific to the release will be used if it exists else release group's one will be used.

If release group is moved to top, users setting a specific release might be surprised to not have matching cover art...

Reply:
> Instead I would just set it to `True` but keep its position. This way cover art specific to the release will be used if it exists else release group's one will be used.
> 
> If release group is moved to top, users setting a specific release might be surprised to not have matching cover art...

I am not sure why it is currently not second - but rather than set it to true and keep it where it, if you are going to leave the Release CAA at the top then I think RG CAA should come immediately after it.

Reply:
This cover art source was initially a hack to get an image when all other sources failed (typically for a release having no cover art on CAA, but one for the release group).
It wasn't meant to become the main source.